src/Microsoft.Docs.Template/Models/GitUserInfo.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal class GitUserInfo
+    public class GitUserInfo
```

Comment:
>public [](start = 4, length = 6)

does this belongs to template model?

Reply:
yes, PageModel uses this for author and contributors

---
In reply to: [210468146](https://github.com/dotnet/docfx/pull/3205#discussion_r210468146) [](ancestors = 210468146)

Reply:
why not put it inside PageModel?

---
In reply to: [210468265](https://github.com/dotnet/docfx/pull/3205#discussion_r210468265) [](ancestors = 210468265,210468146)

Reply:
I thought that's a output `page model`....

---
In reply to: [210468351](https://github.com/dotnet/docfx/pull/3205#discussion_r210468351) [](ancestors = 210468351,210468265,210468146)